frontend/src/metabase/admin/settings/components/EmbeddingSettings/EmbeddingSdkSettings/EmbeddingSdkSettings.tsx

```diff
@@ -108,7 +108,7 @@ export function EmbeddingSdkSettings({
             key="switch-metabase-binaries"
             href={switchMetabaseBinariesUrl}
           >
-            switch Metabase binaries
+            {t`switch Metabase binaries`}
```

Comment:
Fragmented string, maybe needs more attention

Reply:
For now, let's leave it. I think it might be too risky to use Markdown for now, because if the translation fails to preserve the syntax, the component will break. I'd rather have a tiny mistranslation in certain languages than a broken component in any languages. Once we have ways of catching bad Markdown syntax in translations, perhaps we can be bolder.